tvb/simulator/plot/mayavi_tools.py

```diff
@@ -46,24 +46,31 @@
 import scipy as sp
 import networkx as nx
 from tvb.basic.logger.builder import get_logger
+from tvb.simulator.lab import connectivity, plot_connectivity
```

Comment:
Importing from lab here is not right (as it might lead to circular dependencies).
Maybe move plot_with_weights next to plot_connectivity function

Reply:
Fixed